jp2_online/templates/tosp_auth/login_form.html

```diff
@@ -0,0 +1,28 @@
+<div class="row text-center">
```

Comment:
Can this file be implemented in the style of the template? here is the download link: https://codeload.github.com/VinceG/Bootstrap-Admin-Theme-3/zip/master